### ISSUE TITLE
Edge smoothing and IE8

### ISSUE THREAD

**Issue by Edminsson** [openseadragon/openseadragon#1063]
I've been playing around with maxZoomLevel and when you start zooming hard sooner or later     getScaleForEdgeSmoothing will get called and then getRenderedContext will be called. The latter does not work when canvas is not supported, like in IE8. I don't know if you still support IE8, but if you do could this be a solution?

```
if (imageZoom > tiledImage.smoothTileEdgesMinZoom && !tiledImage.iOSDevice && $.supportsCanvas) 
```

**Comment by avandecreme** [comment]
Good catch! That looks like a good fix however I wonder if we have the same sort of issue if the following evaluate to true:
```javascript
    var useSketch = tiledImage.opacity < 1 ||
        (tiledImage.compositeOperation &&
            tiledImage.compositeOperation !== 'source-over') ||
(!tiledImage._isBottomItem() && tile._hasTransparencyChannel());
```
And what if clipping and place holding are used?

**Comment by Edminsson** [comment]
Thanx. Are you referring to useSketch being used in _getContext? The calls to _getContext seem pretty well covered in this aspect aside from the one in getCanvasSize, but the proposed fix should take care of that.

**Comment by avandecreme** [comment]
Ha yes you are right. Would you like to submit a PR for this?

**Comment by Edminsson** [comment]
sure

**Comment by iangilman** [comment]
Fixed by #1064
